### DISCUSSION TITLE
https://eastsea.page/

### DISCUSSION THREAD

**Discussion by giscus** [sdk-xyz/blog-comments#11] [General]
# https://eastsea.page/

https://eastsea.page/

**Comment by Neoncho** [comment]
한국에는 친일파가 진짜 넘쳐납니다.
이 놈들 싸그리 잡아다가 죽창으로 사형시켜야 됨.

**Comment by kiss7kr** [comment]
< 한국어 - Korean >

▶ "동해"는 무엇인가? : 바다의 이름입니다. (예) 영국과 유럽 사이의 "북해"
▶ "동해"의 위치는? : 아시아의 대한민국과 일본 사이에 있습니다.
▶ "동해"의 뜻 : 대륙의 동쪽에 있기 때문에 동해라고 부릅니다. 
(예)대륙중심표기법에서는 바다의 서쪽에 있는 국가의 지명을 바다이름으로 사용합니다. 예를 들어, 서쪽에 러시아의 오호츠크가 있기 때문에 "오호츠크해"라는 바다 이름이 생겼습니다.

**Comment by kiss7kr** [comment]
< English - OFFICIAL >

▶ What is the "East Sea"? : It is the name of a sea. (e.g., The "North Sea" between the United Kingdom and Europe)
▶ Where is the "East Sea" located? : It is situated between South Korea and Japan in Asia.
▶ Meaning of "East Sea" : It is called the East Sea because it is located to the east of the continent.
(e.g., In continent-centric notation, the names of seas are often derived from the location of countries to the west of the sea. For example, the "Sea of Okhotsk" was named after Okhotsk, Russia, which is located to the west of the sea.)

**Comment by kiss7kr** [comment]
< 中國語 - Chinese >

▶ 什么是 "东海"？(例如，英国和欧洲之间的 "北海"）。
▶ 东海 "位于哪里？ 位于亚洲的韩国和日本之间。
▶ 东海 "的含义：因为位于大陆的东面，所以叫东海。
(例如，在以大陆为中心的记法中，海的名称通常取自该海以西国家的位置。例如，"鄂霍次克海 "就是以位于该海西部的俄罗斯鄂霍次克命名的）。

**Comment by kiss7kr** [comment]
< 日本語 - Japanese>

▶ 「東海」とは何ですか？ : 海の名前です。 例）イギリスとヨーロッパの間の「北海」。
▶ 「東海」の位置は？：アジアの大韓民国と日本の間にあります。
▶ "東海"の意味：大陸の東側にあるので東海と呼びます。
(例）大陸中心表記法では、海の西側にある国の地名を海の名前として使います。例えば、西側にロシアのオホーツクがあるため、「オホーツク海」という海の名前が生まれました。

**Comment by kiss7kr** [comment]
< Español  - Spanish >

▶ ¿Qué es el "Mar del Este"? : Es el nombre del mar; (por ejemplo, "Mar del Norte" entre el Reino Unido y Europa)
▶ ¿Dónde se encuentra el "Mar del Este"? : Está entre Corea del Sur y Japón, en Asia.
▶ ¿Qué significa "Mar del Este"? : Se llama Mar del Este porque está en el lado este del continente. 
(Ejemplo) Convenciones de notación geocéntrica : el nombre del país situado en el lado oeste del mar se utiliza como nombre del mar. Por ejemplo, el Mar de Okhotsk se llama "Mar de Okhotsk" porque Okhotsk, Rusia, se encuentra al oeste.

**Comment by kiss7kr** [comment]
< français - French >

▶ Qu'est-ce que la "mer de l'Est" : c'est le nom de la mer (par exemple, la "mer du Nord" entre le Royaume-Uni et l'Europe).
▶ Où se trouve la "mer de l'Est" : entre la Corée du Sud et le Japon, en Asie.
▶ Que signifie "mer de l'Est" : On l'appelle la mer de l'Est parce qu'elle se trouve à l'est du continent. 
(Exemple) Dans le système géocentrique, le nom du pays situé à l'ouest de la mer est utilisé comme nom de la mer. Par exemple, la mer d'Okhotsk est appelée "mer d'Okhotsk" parce qu'Okhotsk, en Russie, se trouve à l'ouest.

**Comment by kiss7kr** [comment]
< Deutsch - German >

▶ Wat is de "Ostmeer": Het is de naam van de zee (bijv. "Noordzee" tussen het Verenigd Koninkrijk en Europa).
▶ Waar ligt de "Ostmeer": Tussen Zuid-Korea en Japan in Azië.
▶ Wat betekent "Ostmeer": Het wordt de Ostmeer genoemd omdat het aan de oostkant van het continent ligt. 
(Voorbeeld) In het geocentrische systeem wordt de naam van het land aan de westkant van de zee gebruikt als de naam van de zee. De Zee van Okhotsk heet bijvoorbeeld "Zee van Okhotsk" omdat Okhotsk, Rusland, in het westen ligt.

**Comment by kiss7kr** [comment]
< Bahasa Indonesia - Indonesian >

▶ Apa itu "Laut Timur"?: Ini adalah nama laut; (misalnya "Laut Utara" di antara Inggris dan Eropa)
▶ Di manakah letak "Laut Timur"?: Laut Timur terletak di antara Korea Selatan dan Jepang di Asia.
▶ Apa arti "Laut Timur": Disebut Laut Timur karena berada di sisi timur benua. 
(Contoh) Dalam sistem geosentris, nama negara di sisi barat laut digunakan sebagai nama laut. Sebagai contoh, Laut Okhotsk dinamai "Laut Okhotsk" karena Okhotsk, Rusia, terletak di sebelah barat.

**Comment by kiss7kr** [comment]
< ру́сский язы́к - Russian >

▶ Что такое "Восточное море"?: Это название моря; (например, "Северное море" между Великобританией и Европой).
▶ Где находится "Восточное море"?: Оно расположено между Южной Кореей и Японией в Азии.
▶ Что означает "Восточное море": Восточное море называется так потому, что оно находится на восточной стороне континента. 
(Пример) В геоцентрической системе в качестве названия моря используется название страны, расположенной на западной стороне моря. Например, Охотское море называется "Охотским", потому что Охотск (Россия) лежит на западе.

**Comment by kiss7kr** [comment]
< Türkçe - Turkish >

▶ "Doğu Denizi" nedir? : Denizin adıdır; (örneğin İngiltere ile Avrupa arasındaki "Kuzey Denizi")
▶ "Doğu Denizi" nerede bulunur?: Asya'da Güney Kore ve Japonya arasındadır.
▶ "Doğu Denizi" ne anlama gelir: Kıtanın doğu tarafında olduğu için Doğu Denizi olarak adlandırılır. 
(Örnek) Jeosentrik sistemde denizin batı tarafında bulunan ülkenin adı denizin adı olarak kullanılır. Örneğin, Okhotsk Denizi "Okhotsk Denizi" olarak adlandırılır çünkü Okhotsk, Rusya, batıda yer alır.

**Comment by kiss7kr** [comment]
< Italiano - Italian >

▶ Che cos'è il "Mare Orientale"? : È il nome del mare (ad esempio, "Mare del Nord" tra il Regno Unito e l'Europa).
▶ Dove si trova il "Mare Orientale"?: Si trova tra la Corea del Sud e il Giappone, in Asia.
▶ Cosa significa "Mare Orientale": Si chiama Mare Orientale perché si trova sul lato orientale del continente. 
(Esempio) Nel sistema geocentrico, il nome del paese che si trova a ovest del mare viene utilizzato come nome del mare. Ad esempio, il Mare di Okhotsk si chiama "Mare di Okhotsk" perché Okhotsk, in Russia, si trova a ovest.

**Comment by kiss7kr** [comment]
< български - Bulgarian >

▶ Какво е "Източно море"? : Това е името на морето (например "Северно море" между Обединеното кралство и Европа).
▶ Къде се намира "Източно море"?: То е между Южна Корея и Япония в Азия.
▶ Какво означава "Източно море": Нарича се Източно море, защото се намира в източната част на континента. 
(Пример) В геоцентричната система за име на морето се използва името на страната, която се намира от западната страна на морето. Например Охотско море е наречено "Охотско море", защото Охотск, Русия, се намира на запад.

**Comment by kiss7kr** [comment]
< čeština - Czech >

▶ Co je to "Východní moře"? : Je to název moře (např. "Severní moře" mezi Spojeným královstvím a Evropou).
▶ Kde se nachází "Východní moře": Nachází se mezi Jižní Koreou a Japonskem v Asii.
▶ Co znamená "Východní moře": Nazývá se Východní moře, protože se nachází na východní straně kontinentu. 
(Příklad) V geocentrickém systému se jako název moře používá název země na západní straně moře. Například Ochotské moře se jmenuje "Ochotské moře", protože Ochotské moře v Rusku leží na západě.

**Comment by sdk-kr** [comment]
North Korea recently raised objections to a German newspaper. The controversy stemmed from an article in "Bild," a leading German daily, that reported on the appearance of Kim Ju-ae, the daughter of North Korean Chairman Kim Jong-un. However, North Korea's outrage was not because the article mentioned the daughter of the revered Chairman, but due to the use of the term "Sea of Japan." When "Bild" reported that "Kim Ju-ae lives on the coast of the Sea of Japan in Wonsan," the North Korean Embassy in Germany retorted, "The accurate name is the East Sea (Korean East Sea)."

![Screenshot 2023-08-22 at 10 40 51 PM](https://github.com/sdk-xyz/blog-comments/assets/16316626/1cdbbf1a-48d4-4da9-87c8-2d596ea04f2a)
https://www.donga.com/news/article/all/20140313/61666228/1

![Screenshot 2023-08-22 at 10 39 30 PM](https://github.com/sdk-xyz/blog-comments/assets/16316626/0cf1efaa-2924-4c92-9655-3c77f85a7fab)
https://www.kado.net/news/articleView.html?idxno=1172844

In 1844, a map produced in Japan, titled "Shinjeyeojijeondo" (with annotations on the right), clearly labels the East Sea as the "Joseon Sea" when the Korean Peninsula is zoomed in on. This image was provided by Kyung Hee University Hyejung Museum.

For the first time in South Korea, an original world map created by a 19th-century Japanese scholar that denotes the East Sea as "Joseon Sea (朝鮮海)" has been unveiled.

On the 12th, the Kyung Hee University Hyejung Museum (Director Kim Hyejung) announced at the Seoul Arts Center in Seocho-gu, Seoul, "In the 'Shinjeyeojijeondo (新製輿地全圖)' made by Mitsukuri Shogo (箕作省吾) in 1844, using a French map as a reference, the East Sea is distinctly labeled as Joseon Sea." The Shinjeyeojijeondo drew significant attention last year when the National Archives released a replica. While the original was exhibited in Mongolia in 2011, this is its first display in South Korea. This map will be showcased to the public starting from the 22nd in a special exhibition titled "The East Sea Through Ancient World Maps" at the Seoul Arts Center Calligraphy Museum.
